### PR TITLE
Ref #49; Moved denied prompt loading to parser

### LINKED ISSUE #49: Move loading of permission denied prompt file to DAEConfigParser

### PR DESCRIPTION
**Pull request by iordanivanov**
Updated unit tests accordingly

closes #49

### CODE REVIEW

File: dae/dae/configuration/dae_config_parser.py [hunk 1]
```diff
@@ -124,4 +124,15 @@ def parse(cls, config, dae_data_dir=None):
                 if config.genomic_scores_db else None,
         }
 
+        if config.gpfjs:
+            filepath = config.gpfjs.permission_denied_prompt
+            assert os.path.exists(filepath), filepath
+            with open(filepath, 'r') as prompt_file:
+                config.gpfjs.permission_denied_prompt = prompt_file.read()
+        else:
```
Comment (nikolaystanishev) [inline dae/dae/configuration/dae_config_parser.py]: You should probably consider moving this to the DEFAULT_SECTION_VALUES.

Comment (iordanivanov) [reply dae/dae/configuration/dae_config_parser.py]: :+1:

Comment (iordanivanov) [reply dae/dae/configuration/dae_config_parser.py]: on second thought, I don't like using the same attribute for both the filepath and the contents of the file; I'll fix it and re-request the review

**Comment by iordanivanov** [comment]
when this is merged into master, we'll need to change data-hg19's DAE.conf

**Pull request merged**